packages/dds/tree/src/test/feature-libraries/editable-tree/editableTree.editing.spec.ts

```diff
@@ -78,23 +78,21 @@ function getTestSchema(fieldKind: FieldKind): SchemaData {
 	);
 }
 
-// TODO: There are two kinds of users of this in this file. Both should be changed:
-// Tests which are testing collaboration between multiple trees should be adjusted to not do that, or moved elsewhere (merge/collaboration is not the focus of this file).
-// Tests which are using a single tree should just use a MockFluidDataStoreRuntime instead of all the complexity of TestTreeProvider.
```

Comment:
I think this still applies to TestTreeProviderLite.
If we remove the multi tree cases, we should be able to just do `const tree = factory.create(new MockFluidDataStoreRuntime(), "test");` in here.

Also longer term we should be able to do even better than that, and just use the branch APIs directly and not depend on the SharedTree class, its factory or fluid at all.

I think your changes here are progress, but not quite enough to remove this todo, just make it less important.